### PR TITLE
routes: rename workflows to workspace

### PR DESCRIPTION
**Pull request by oliver-sanders**
> As mentioned [on Element](https://matrix.to/#/!JpEnWGPfuvHyztRhmT:matrix.org/$167171187833693iatcO:matrix.org?via=matrix.org)

* Rename the `#/workflows` route to `#/workspace`.
* Split the GScan component into a view and component.
* Use the `#/workflows` route for GScan.
* Remove the sidebar for the standalone views.

Ideally get this into 1.4.0 before this gets into peoples browser history.

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Does not need tests (why?).
- [ ] Appropriate change log entry included.
- [x] No documentation update required.

### CODE REVIEW

File: src/components/cylc/gscan/GScan.vue [hunk 1]
```diff
@@ -200,38 +200,38 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
...
 import uniq from 'lodash/uniq'
-import subscriptionComponentMixin from '@/mixins/subscriptionComponent'
 import TaskState from '@/model/TaskState.model'
-import SubscriptionQuery from '@/model/SubscriptionQuery.model'
 import { WorkflowState } from '@/model/WorkflowState.model'
 import Job from '@/components/cylc/Job'
 import Tree from '@/components/cylc/tree/Tree'
 import WorkflowIcon from '@/components/cylc/gscan/WorkflowIcon'
 import { filterHierarchically } from '@/components/cylc/gscan/filters'
-import { GSCAN_DELTAS_SUBSCRIPTION } from '@/graphql/queries'
```
Comment (oliver-sanders) [inline src/components/cylc/gscan/GScan.vue:214]: This separates the component from the view code as already done for some of the other views.

* The component should be something you can mount by itself.
  * Functional dependencies should provided via `props`.
  * It shouldn't issue subscriptions or go looking in the store, it should work by itself.
  * It should be generic-ish and reusable.
  * It should be component testable!
* The "view" should contain the application logic.
  * It should start the subscription (via the required mixins).
  * It should fetch the relevant items from the store.
  * It should be functional testable.

File: src/router/paths.js [hunk 2]
```diff
@@ -57,13 +57,48 @@ export default [
...
+
+  // the standalone views
+  {
+    path: '/workflows',
+    name: i18n.t('App.workflows'),
+    view: 'Workflows',
+    meta: {
+      layout: 'default',
+      toolbar: false,
+      showSidebar: false
```
Comment (oliver-sanders) [inline src/router/paths.js:91]: Don't show the sidebar in the workflows view, it would be duplication.

**Comment by oliver-sanders** [comment]
> Firefox e2e test failure fixed in another PR.
>    1) Table view
       Filters
         displays and sorts dt-mean:
     CypressError: Timed out retrying after 10050ms: `cy.click()` failed because this element:
      `<path d="M13,20H11V8L5.5,13.5L4.08,12.08L12,4.16L19.92,12.08L18.5,13.5L13,8V20Z"></path>`

**Comment by MetRonnie** [comment]
Sorry if already discussed, but what about `workflow` (singular)_instead of `workspace`?

**Comment by oliver-sanders** [comment]
Workflow is a little ambiguous as all of the routes are "workflow" views. We do also intend to use the workspace view to show multiple workflows.

**Review by wxtim — APPROVED** [review]
Legit.
Happy for you to merge if you're ok to break master, or to wait for the fix to the broken test to go in first.

**Comment by oliver-sanders** [comment]
Rebased and applied the `force` fix to the failing e2e test...

**Pull request merged**